apps/Gruntfile.js

```diff
@@ -125,6 +127,12 @@ module.exports = function (grunt) {
         },
         {
           expand: true,
+          cwd: piskelRoot,
+          src: '**',
+          dest: 'build/package/js/piskel/'
```

Comment:
They are already minified via Piskel's build process.

As-we-type I am working out a nice way to run debug Piskel inside gamelab (it's not as easy as doing a debug build) but that shouldn't block this change.